documentation/resources/rbenv_gem.md

```diff
@@ -0,0 +1,34 @@
+# Gem
+
+Used to install a gem into the selected rbenv environment.
+
+## Properties
+
+| Name                   | Type              | Default              | Description |
+| ---------------------- | ----------------- | -------------------- | ----------- |
+| clear_sources          |                   | `true, false`        |             |
```

Comment:
I believe you meant to put the "default" as type and then add a proper default value here.

Also, all of the proper names should appear as "tech text" IMO in all the docs.